### PR TITLE
DOC-2450: Restructure editor-plugin-version.adoc

### PR DESCRIPTION
**Pull request by FarzadHayat**
Ticket: DOC-2450

Site: [Staging branch](http://docs-hotfix-7-doc-2450.staging.tiny.cloud/docs/tinymce/latest/editor-plugin-version)

**HOTFIX**

Changes:
* Add examples of specifying minor and patch versions
* Move the "Difference between using TinyMCE 7, 7-Testing, and 7-Dev release channels" section to the top of the document so that information about specifying editor version is all in one place.

Pre-checks:
- [x] Branch prefixed with `feature/<version>/`, `hotfix/<version>/`, `staging/<version>/`, or `release/<version>/`
- [-] `modules/ROOT/nav.adoc` has been updated `(if applicable)`
- [-] Files has been included where required `(if applicable)`
- [-] Files removed have been deleted, not just excluded from the build `(if applicable)`
- [-] Files added for `New product features`, and included a `release note` entry.

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

File: modules/ROOT/pages/editor-plugin-version.adoc
Comment (kemister85) [inline modules/ROOT/pages/editor-plugin-version.adoc:136]:
```suggestion
<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/7/cloud-plugins.min.js?mentions&powerpaste&advcode" referrerpolicy="origin"></script>
```

File: modules/ROOT/pages/editor-plugin-version.adoc
Comment (kemister85) [inline modules/ROOT/pages/editor-plugin-version.adoc:120]:
```suggestion
<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/7/plugins.min.js?mentions=sdk&powerpaste=sdk" referrerpolicy="origin"></script>
```

File: modules/ROOT/pages/editor-plugin-version.adoc
Comment (kemister85) [inline modules/ROOT/pages/editor-plugin-version.adoc:111]:
```suggestion
<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/7/plugins.min.js" referrerpolicy="origin"></script>
```

**Review by kemister85 — APPROVED** [review]
@FarzadHayat If we publish these changes, we will need to update the tinymce/6 docs page also to reflect the same content.
- [x] Follow-up ticket created for `hotfix/6` docs editor-plugin-versions.adoc` page.

**Review by ShiridiGandham — APPROVED** [review]
LGTM - https://www.diffchecker.com/71u6dcsl/

**Comment by FarzadHayat** [comment]
> @FarzadHayat If we publish these changes, we will need to update the tinymce/6 docs page also to reflect the same content.
> 
> * [ ]  Follow-up ticket created for `hotfix/6` docs editor-plugin-versions.adoc` page.

https://github.com/tinymce/tinymce-docs/pull/3340

**Comment by kemister85** [comment]
@FarzadHayat Whilst i dont disagree with these new changes from the latest commits, the general practice is to **not** add additional changes the context on a already approved PR, specifically one that is a high traffic page such as this one.

**Comment by FarzadHayat** [comment]
> @FarzadHayat Whilst i dont disagree with these new changes from the latest commits, the general practice is to **not** add additional changes the context on a already approved PR, specifically one that is a high traffic page such as this one.

You're right. I shouldn't have made further changes after receiving approvals. I will re-request approvals to ensure the new changes have been reviewed.

**Review by kemister85 — APPROVED** [review]
LGTM

**Pull request merged**
